lisa/operating_system.py

```diff
@@ -1034,6 +1034,24 @@ def _package_exists(self, package: str) -> bool:
 
         return False
 
+    def install_epel(self) -> None:
+        major = self._node.os.information.version.major
+        assert_that(major).described_as(
+            "Fedora/RedHat version must be greater than 7"
+        ).is_greater_than_or_equal_to(7)
+        epel_release_rpm_name = f"epel-release-latest-{major}.noarch.rpm"
+        self._install_packages(
+            [f"https://dl.fedoraproject.org/pub/epel/{epel_release_rpm_name}"]
+        )
+        self._node.execute(
+            "sudo yum update -y --disablerepo='*' --enablerepo='*microsoft*'",
```

Comment:
you want to check epel works well using this command? why only enable microsoft repo?
how about removing epel repo if epel doesn't work on current VM?

Reply:
I had encountered some issue during testing where the regular yum update wasn't working, it must have been some other issue though since regular yum update seems to work fine now that I'm testing this morning. I'll adjust this.